### DISCUSSION TITLE
i am beginner  in this project. can anyone tell me how setup this project locally.

### DISCUSSION THREAD

**Discussion by rahulkiller104** [CGAL/cgal#5734] [Q&A] [answered]
what are prerequisite?
what are the steps for setup locally?

**Comment by lrineau** [comment, marked as answer]
Everything is explained in the [Getting Started chapter of the manual](https://doc.cgal.org/latest/Manual/general_intro.html).

Prerequisites are in the section [Essential and Optional Third Party Dependencies](https://doc.cgal.org/latest/Manual/thirdparty.html). And the steps depends on you platform:
  - [for Linux and MacOS](https://doc.cgal.org/latest/Manual/usage.html) (both POSIX systems),
  - [for Microsoft Windows](https://doc.cgal.org/latest/Manual/windows.html).

**Reply by rahulkiller104** [reply]
thaks for replying
